mmdet/models/dense_heads/atss_head.py

```diff
@@ -9,6 +9,8 @@
 from ..builder import HEADS, build_loss
 from .anchor_head import AnchorHead
 
+eps = 1e-12
```

Comment:
We'd better use EPS to avoid conflict in case there some places also use eps as a variable.